Modified the errors produced by debugLauncher.js when launch.json is not configured right (#14739)
microsoft/vscode-python: news/14739.md

> Modified the errors generated by debugLauncher.ts to be more specific when launch.json is not properly configured

Reviewer: ```suggestion
Modify errors generated when `launch.json` is not properly configured to be more specific about which fields are missing.
```
Making the message user friendly as users have no idea about `debugLauncher.ts`.
